apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -294,6 +298,10 @@ export default function Rundown({ data }: RundownProps) {
                 lastEvent = thisEvent;
 
                 if (isPlayableEvent(entry)) {
+                  if (!isPast) {
+                    //TODO: how to handle overlaps
+                    accumulatedGap = calculateAccumulatedGap(accumulatedGap, entry, lastEvent);
```

Comment:
question: is this not coming from the other PR?

Reply:
yes patially